.github/workflows/ci.yml

```diff
@@ -26,3 +33,33 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: make ci
+
+  cypress-e2e:
+    name: Cypress Emulator Tests for Flow-Playground
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    container: cypress/browsers:node16.14.2-slim-chrome103-ff102
```

Comment:
Couldn't we use an existing cypress actions as seen here https://docs.cypress.io/guides/continuous-integration/github-actions#Testing-in-Chrome-and-Firefox-with-Cypress-Docker-Images 
?